src/app/pages/MapPage/query-params.test.ts

```diff
@@ -0,0 +1,29 @@
+import { panoTagParam } from './query-params'
+
+describe('Map Query Parameters', () => {
+  describe('Panorama', () => {
+    describe('tags', () => {
+      it('should encode the value to parameter value', () => {
```

Comment:
Can we agree to use verbs here instead of 'should'? I would prefer this for example:
```suggestion
      it('encodes the value to parameter value', () => {
```

What do you guys think?

Reply:
Agree

Reply:
`should` Implies may or may not.